Testing/TestSignalSlot.cc

```diff
@@ -8,13 +8,18 @@
 #include "Signal.hh"
 #include "Slot.hh"
 
+#include "TestControllerClasses.hh"
+
 #include "catch.hpp"
 
 
 TEST_CASE( "signal_slot", "[signal],[slot],[processor]" )
 {
     using namespace Nymph;
 
+    // need a controller to exist
+    CIQThrowController controller;
+
```

Comment:
I might be misunderstanding part of the structure here, but why does this controller object need to exist? Is this just a placeholder for future work? I'm not quite sure how this fits into this test. This controller object appears in a few other places as well, but I'm just making one comment here for them.

Reply:
`ControlAccess` needs to point to a real Controller to function properly.  Both `Signal` and `Slot` use `ControlAccess`, and therefore we need a real Controller to exist in this test case.

Reply:
This could potentially be mocked up in different ways, but just instantiating one of the test controllers works well enough for that purpose.